gapil/bapi/bapi.proto

```diff
@@ -19,77 +19,163 @@ option go_package = "github.com/google/gapid/gapil/bapi";
 
 message Content {
   Instances instances = 1;
-  repeated uint64 apis = 2;  // API ID
+  Mappings mappings = 2;
+  repeated uint64 apis = 3;  // API ID
 }
 
 message Instances {
-  repeated API api = 1;
-  repeated Block block = 2;
-  repeated Callable callable = 3;
-  repeated Class class = 4;
-  repeated Definition definition = 5;
+  repeated Abort abort = 1;
+  repeated Annotation annotation = 2;
+  repeated API api = 3;
+  repeated ArrayAssign array_assign = 4;
+  repeated ArrayIndex array_index = 5;
+  repeated ArrayInitializer array_init = 6;
+  repeated Assert assert = 7;
+  repeated Assign assign = 8;
+  repeated BinaryOp binary_op = 9;
+  repeated BitTest bit_test = 10;
+  repeated Block block = 11;
+  repeated BoolValue bool_value = 12;
+  repeated Branch branch = 13;
+  repeated Call call = 14;
+  repeated Callable callable = 15;
+  repeated Case case = 16;
+  repeated Cast cast = 17;
+  repeated Choice choice = 18;
+  repeated Class class = 19;
+  repeated ClassInitializer class_init = 20;
+  repeated Clone clone = 21;
+  repeated Copy copy = 22;
+  repeated Create create = 23;
+  repeated DeclareLocal declare_local = 24;
+  repeated Definition definition = 25;
+  repeated DefinitionUsage definition_usage = 26;
   repeated Enum enum
```

Comment:
What is going on here? Is it because we have an enum named enum?

Reply:
Yup, it is. I could rename it if you want - but would like to do so in a follow up CL.